include/opendht/network_engine.h

```diff
@@ -375,7 +399,7 @@ class NetworkEngine final
     static const constexpr size_t NODE4_INFO_BUF_LEN {26};
     /* the length of a node info buffer in ipv6 format */
     static const constexpr size_t NODE6_INFO_BUF_LEN {38};
-    /* TODO */
+    /* the maximum time we wait for an UDP reply */
```

Comment:
from https://linux.die.net/man/2/sendto :

>  MSG_CONFIRM
>  -  Tell the link layer that forward progress happened: you got a successful reply from the other side. If the link layer doesn't get this it will regularly reprobe the neighbor (e.g., via a unicast ARP). Only valid on SOCK_DGRAM and SOCK_RAW sockets and currently only implemented for IPv4 and IPv6. See arp(7) for details. 

We set this flag if we got a message from the node in the last UDP_REPLY_TIME.